### ISSUE TITLE
Callout card illustration is missing alt text

### ISSUE THREAD

**Issue by matt-neill** [smile-io/ember-polaris#267]
## Problem
Missing `alt` text in the `<img />` tag of the [callout card](https://github.com/smile-io/ember-polaris/blob/cadfda66d08c0e89a711e9b2b4b3a4f5d58d7f10/addon/templates/components/polaris-callout-card.hbs#L39) (line 39)

## Solution
Add some alt text, yo

**Comment by andrewpye** [comment]
@matt-neill closing this as this is the aim of `ember-polaris` is to match the behaviour of the Shopify React implementation in Ember-land, and their implementation of this also has a hard-coded [empty `alt` attribute](https://github.com/Shopify/polaris-react/blob/master/src/components/CalloutCard/CalloutCard.tsx#L58) on the image.

**Comment by vladucu** [comment]
@andrewpye what do you think of opening an issue/PR for this with Shopify?

**Comment by andrewpye** [comment]
Could do... waiting to hear from @matt-neill what the use case was first. I guess Shopify had their reasons for setting a blank `alt` value when they originally built the component 🤷‍♂️
